share/x-osu.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0"?>
+<mime-info xmlns='http://www.freedesktop.org/standards/shared-mime-info'>
+  <mime-type type="text/x-osu">
+    <comment>osu! beatmap file</comment>
+    <comment lang="fr">Fichier de beatmap osu!</comment>
+    <magic priority="50">
+      <match type="string" offset="0" value="osu file format"/>
+    </magic>
+    <glob pattern="*.osu"/>
+  </mime-type>
+</mime-info>
```

Comment:
I know I'm the one that suggested `x-osu`, but since there are other file types for osu!, rather than naively name them x-osb, x-osk, maybe x-osu-something is better.

Here, that would be `text/x-osu-beatmap`. What do you think?

You're right for text/ vs application/, application is binary according to [MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types).

Reply:
Almost there! The file should be renamed x-osu-beatmap.xml